distribution/openrct2.d.ts

```diff
@@ -3100,14 +3100,14 @@ declare global {
         staffType: StaffType;
 
         /**
-         * Colour of the staff member. Not applicable for entertainers.
+         * Colour of the staff member. Not applicable to entertainers.
          */
         colour: number;
 
         /**
-         * The entertainer's costume, only applicable for entertainers.
+         * The staff member's costume.
```

Comment:
I'm missing `availableCostumes` here in the declaration file?

Reply:
Done!